hydromt/models/model.py

```diff
@@ -1181,66 +976,6 @@ def write_geoms(
                 gdf = gdf.to_crs(4326)
             gdf.to_file(_fn, **kwargs)
 
-    @property
-    def staticgeoms(self):
```

Comment:
Same here link this PR to the deprecated staticgeoms issue

Reply:
I've linked it below. Good point.